kedro-docker/features/steps/cli_steps.py

```diff
@@ -161,6 +161,35 @@ def create_project_from_config_file(context):
     # add a consent file to prevent telemetry from prompting for input during e2e test
     telemetry_file = context.root_project_dir / ".telemetry"
     telemetry_file.write_text("consent: false", encoding="utf-8")
+
+    # override base logging configuration to simplify assertions
+    logging_conf = context.root_project_dir / "conf" / "base" / "logging.yml"
+    logging_conf.write_text(
+        """
+version: 1
+
+disable_existing_loggers: False
+
+formatters:
+  simple:
+    format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: INFO
+    formatter: simple
+    stream: ext://sys.stdout
+
+loggers:
+  kedro:
+    level: INFO
+
+root:
+  handlers: [console]
+"""
```

Comment:
I think the indentation here broke, I feel this should be inline with the opening `"""`.

Reply:
Good point, I remember trying this method but something dind't work. I will have another look